### PR TITLE
adding a new open api for weather

### PR DESCRIPTION
**Pull request by grepit**
@davemachado  would you please review this when you have few minutes thanks .

### CODE REVIEW

**Review by grepit — COMMENTED** [review]
please review this

**Review by grepit — COMMENTED** [review]
fixed the formatting and added the link again

**Review by grepit — COMMENTED** [review]
everything passed please approve

**Comment by grepit** [comment]
@yannbertrand @davemachado @pyaillet , would one of you please review this CR and approve for merging .
I know you guys are busy but love to see this CR getting merged :) Thanks in advance

File: README.md [hunk 1]
```diff
@@ -927,6 +927,7 @@ API | Description | Auth | HTTPS | CORS |
 | [OpenUV](https://www.openuv.io) | Real-time UV Index Forecast | `apiKey` | Yes | Unknown |
 | [OpenWeatherMap](https://openweathermap.org/api) | Weather | `apiKey` | Yes | Unknown |
 | [Storm Glass](https://stormglass.io/) | Global marine weather from multiple sources | `apiKey` | Yes | Yes |
+| [US Weather ](https://api.weather.gov/points/47.7601,-122.2054) | Weather for US by Latitude and Longitude | `apiKey` | Yes | Unknown |
```
Comment (marekdano) [inline README.md:930]: Is `apiKey` required to use this API? It doesn't look like, I can call the API with no API key.

**Review by marekdano — COMMENTED** [review]
Thanks for the contribution @grepit!
I left a comment for `apiKey`.

Can I ask you to resolve the conflict and squash all your commits, please?

**Comment by matheusfelipeog** [comment]
This PR is inactive, so the API was added in another PR/commit.

More information at: #2002

**Pull request closed without merging**